release notes/v0.49.0.md

```diff
@@ -0,0 +1,202 @@
+k6 `v0.49.0` is here 🎉! This release includes:
+
+- Add a built-in [web dashboard](https://grafana.com/docs/k6/latest/results-output/web-dashboard/) that displays test results in real time.
+- Add `clear` functionality to the browser module's `locator` classes.
+- Merge the gRPC experimental module back into the gRPC core module.
+- Add the ability to get the selection from an element in `k6/html`. 
+- Collect internal modules and outputs used by a script.
+- Get `k6/experimental/timers` ready to be stabilized.
+
+
+## Breaking changes
+
+- [#3494](https://github.com/grafana/k6/pull/3494) stops updating `loadimpact/k6` docker image. If you still use it, please migrate to the [grafana/k6](https://hub.docker.com/r/grafana/k6) image.
+- [browser#1111](https://github.com/grafana/xk6-browser/pull/1111) removes `timeout` option for `isVisible` and `isHidden` since the API no longer waits for the element to appear on the page.
+
+## New features
+
+### Web Dashboard
+
+The new [web dashboard](https://grafana.com/docs/k6/latest/results-output/web-dashboard/) brings real-time visualization to load testing. This feature allows users to monitor test progress and analyze
+results dynamically, enhancing the overall testing experience.
+
+#### Real-time test results
+
+Activate this feature using the environment variable `K6_WEB_DASHBOARD=true`. For this initial release, the dashboard is not enabled by default to allow users to opt into this new experience as it evolves.
+
+```bash
+K6_WEB_DASHBOARD=true k6 run script.js
+```
+
+Once enabled and the test script is running, navigate to [http://localhost:6565](http://localhost:6565) in your web browser to access the dashboard.
+
+![k6 Web Dashboard Overview](https://github.com/grafana/xk6-dashboard/blob/master/screenshot/k6-dashboard-overview-light.png?raw=true)
+
+#### Test report
+
+The web dashboard also offers an [HTML test report](https://github.com/grafana/xk6-dashboard/blob/master/screenshot/k6-dashboard-html-report-screen-view.png?raw=true) for detailed analysis, enabling easy sharing and downloading capabilities for
```

Comment:
Maybe explicitly mention that the external link is an example, but if it had a documentation page, that could be better.

```suggestion
The web dashboard also offers an HTML test report (see [an example](https://github.com/grafana/xk6-dashboard/blob/master/screenshot/k6-dashboard-html-report-screen-view.png?raw=true)) for detailed analysis, enabling easy sharing and downloading capabilities for
```